test/Project.toml

```
@@ -6,11 +6,18 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StartUpDG = "472ebc20-7c99-4d4b-9470-8fde4e9faa0f"
+StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Triangulate = "f7e6ffb2-c36d-4f8f-a77e-16e897189344"
 
 [compat]
 Cassette = "0.3.5"
 ForwardDiff = "0.10"
 MPI = "0.16, 0.17, 0.18"
-OrdinaryDiffEq = "5.60"
-Plots = "1.9"
+Plots = "1.9, 1.16"
```

This is equivalent to `Plots = "1.9"` (since it's only a minor version bump). Since you added it, I expect there is some reason to use the higher version, so let's just do this.
```suggestion
Plots = "1.16"
```